examples/basics/batches.ipynb

```diff
@@ -1,354 +1,434 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    project = client.create_project( name="Demo-Batches-Project",                                 
</pre><p>Maybe link to queue management tutorial here. because this is highly related.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/750/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>